actions_test.go

```diff
@@ -134,11 +132,14 @@ func TestCreateTriggerDefinition(t *testing.T) {
 
 	// Act
 	trigger, err := client.CreateTriggerDefinition(ol.CustomActionsTriggerDefinitionCreateInput{
-		Name:        "Deploy Rollback",
-		Description: ol.NewString("Disables the Deploy Freeze"),
-		Action:      *newID,
-		Owner:       *newID,
-		Filter:      ol.NewID("987654321"),
+		Name:                   "Deploy Rollback",
+		AccessControl:          ol.NewEnum[ol.CustomActionsTriggerDefinitionAccessControlEnum](ol.CustomActionsTriggerDefinitionAccessControlEnumEveryone),
+		Description:            ol.NewString("Disables the Deploy Freeze"),
+		ResponseTemplate:       ol.NewString(""),
```

Comment:
Passing a pointer to an empty string feels weird, why are we doing that in the test?

Reply:
I figure you're doing this because the field was changed to a pointer just now and the original mocked response JSON passed an empty string,

In that case it can be considered out of scope to fix this. 👍